templates/convoy-efs/0/Readme.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Can you either add a description here or just delete this file? Would rather not have an empty readme